wolox-books/src/app/services/book-service.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Book } from '../models/book.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BookServiceService {
+  readonly URL_API = 'https://wbooks-api-stage.herokuapp.com/api/v1/';
+  routeBook = 'books';
```

Comment:
routeBook should be "baseUrl + 'books'", all the endpoint that will be used on this service will use the same URL